ci/build-and-test-bindings.dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM pierrpebay/vt-tv:master AS build
```

Comment:
We should probably shouldn't use your account here. Do you have access to push to the main one?

Reply:
Let's ignore this issue for now and fix in in the follow-on issue for implementing full CI.